woodwork/table_accessor.py

```diff
@@ -722,7 +718,7 @@ def _get_invalid_schema_message(dataframe, schema):
             return f'dtype mismatch for column {name} between DataFrame dtype, '\
                 f'{df_dtype}, and {schema.logical_types[name]} dtype, {schema_dtype}'
     if schema.index is not None:
-        if not all(dataframe.index == dataframe[schema.index]):
+        if not all(dataframe.index == dataframe[schema.index]):  # --> not rel for dask bc no underlying and isunique nto going to happen
```

Comment:
Remove comment?

Reply:
Or we can fix when things get merged with the Dask PR...

Reply:
oh haha I'll remove. Was something I was thinking about for your Dask PR, but I realized I liked the way you handled it better.